3주차/[Lv.2]_튜플/최원석.js

```diff
@@ -0,0 +1,5 @@
+function solution(s) {
+  return JSON.parse(s.replace(/\{|\}/g, (match) => (match === '{' ? '[' : ']')))
+    .sort((a, b) => a.length - b.length)
+    .reduce((acc, cur) => [...acc, ...cur.filter((v) => !acc.includes(v))], []);
```

Comment:
reduce도 정말 잘 활용하시네요.... 대박입니다... 👍 
근데 원석님 코드 보고 영감이 떠올라서 나름 수정을 해봤습니다! 

```suggestion
.reduce((acc, cur) => [...new Set([...acc, ...cur])], []);
```
속도도 조금 더 빨라지네요! 그냥 참고만 해주세요!!

before
![image](https://github.com/prgrms-web-devcourse/FEDC4-JSAlgorithm-study/assets/113358733/995d0d6c-0dee-4d4a-8443-05d7be896949)

after
![image](https://github.com/prgrms-web-devcourse/FEDC4-JSAlgorithm-study/assets/113358733/dd14e998-7fe5-428d-894f-4c19324c44e4)